app/views/shared/_display_ad.html.erb

```diff
@@ -4,7 +4,12 @@
   data-category-impression="<%= DisplayAdEvent::CATEGORY_IMPRESSION %>"
   data-context-type="<%= data_context_type %>">
   <div class="crayons-sponsorship__header relative">
-    <div class="crayons-sponsorship__title"><%= Settings::Community.community_name %></div>
+    <% if Page.exists?(slug: "billboards") && display_ad.organization_id? && display_ad.type_of == "community" %>
```

Comment:
Is `Page.exists?(slug: "billboards")` necessary in this context? It seems to add an extra query we probably don't need for this logic, eh?